AboutMe.Web/Controllers/AccountController.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Application.Dtos.Identity;
+using Core.Constants;
+using Core.Entities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AboutMe.Web.Controllers
+{
+    [AllowAnonymous]
+    public class AccountController : Controller
+    {
+        private readonly SignInManager<User> _signInManager;
+        private readonly UserManager<User> _userManager;
+
+        public AccountController(
+            UserManager<User> userManager,
+            SignInManager<User> signInManager)
```

Comment:
```suggestion
            SignInManager<User> signInManager
)
```
try to keep consistent formatting, in the future if you wrap parenthesis on another line it will be easier to add new dependencies by just adding a new line. Just a personal preference.